src/compiler/diagnosticMessages.json

```diff
@@ -1344,7 +1344,7 @@
         "category": "Error",
         "code": 2375
     },
-    "A 'super' call must be the first statement in the constructor when a class contains initialized properties or has parameter properties.": {
+    "A 'super' call must be the first statement in the constructor when a class contains initialized properties, parameter properties, or downleveled private identifiers.": {

```

Comment:
Removed `downleveled` because we currently move private identifier initializers to the constructor even if we are not downleveling (to preserve evaluation order with TS class fields).